docs/reference/src/language/concepts/focus.md

```diff
@@ -1,10 +1,11 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # Focus Handling
 
-Certain elements such as `TextInput` accept not only input from the mouse/finger but
+Certain elements such as `TextInput` accept input from the mouse/finger and
 also key events originating from (virtual) keyboards. In order for an item to receive
-these events, it must have the focus. This is visible through the `has-focus` (out) property.
-
+these events, it must have focus. This is visible through the `has-focus` (out) property.
+<!-- (out)? -->
```

Comment:
Out means that it's a property that cannot be set by the user, but it can only be read. The focusable element here controls when `has-focus` is true. There are also `in` properties (like write-only) and `in-out` (read-write).

Reply:
OK, is this standard rust or standard Slint?

Reply:
Since this refers to `has-focus` and properties in Slint, this is part of the Slint language's way of declaring properties.

Reply:
Ok, will leave for now and maybe link up related concepts as a future task